autotest/ogr/ogr_mitab.py

```diff
@@ -1907,17 +1824,16 @@ def test_ogr_mitab_43():
             f_ref.DumpReadable()
             pytest.fail()
 
-    gdaltest.mapinfo_drv.DeleteDataSource('/vsimem/all_geoms_block_512.tab')
-    gdaltest.mapinfo_drv.DeleteDataSource('/vsimem/all_geoms_block_32256.tab')
+    ogrtest.mapinfo_file_drv.DeleteDataSource('/vsimem/all_geoms_block_512.tab')
```

Comment:
I renamed these all because they're set up automatically by lower-casing the driver name and replacing spaces with underscores; see the implementation of `require_driver` in `conftest.py`